src/indieweb_utils/replies/context.py

```diff
@@ -75,11 +77,10 @@ def _process_h_entry_author(h_entry: dict, url: str, domain: str) -> Tuple[str,
         try:
             author = mf2py.parse(requests.get(author_url, timeout=10, verify=False).text)
         except requests.exceptions.RequestException:
-            author = {}
-
-        h_card = [item for item in author["items"] if item.get("type", []) == ["h-card"]][0]
-
-        author_url, author_name, author_image = _get_author_properties(author_url, h_card)
+            pass
```

Comment:
In the case of a network error, the previous code would result in a KeyError ( the `author` dict is empty and has no key `"items"`.  This changes the flow so we only over call `_get_author_properties` if we don't have a network error.